app/product/friendly-url-redirect.controller.js

```diff
@@ -28,11 +28,12 @@ module.exports = async function redirectToProduct (req, res) {
     logger.info(`Redirecting to ${payUrl}`)
     return res.redirect(payUrl)
   } catch (err) {
-    if (err.errorCode && err.errorCode >= 500) {
+    if (err.errorCode === 404) {
       logger.error(`Error getting product: ${err.message} errorCode=${err.errorCode}`)
```

Comment:
We won't want to log at error level if the error is a 404 as this isn't an error we care about.

However, do we want to continue logging at error level for a 5xx?

Reply:
Is there any harm logging the 404?

I'm not sure it's a hard and fast rule to log for a 5xx either... we don't in [resolve-product](https://github.com/alphagov/pay-products-ui/blob/master/app/middleware/resolve-product.js#L33). However for this PR, a 5xx would be logged in the else statement a few lines below.

Reply:
No, you're right...a 404 isn't an error. I'll remove that log.